### DISCUSSION TITLE
TLS certificate is being provisioned error.

### DISCUSSION THREAD

**Discussion by flyingcards** [community/community#102872] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I always get this error with the use of my custom domain:

![image](https://github.com/community/community/assets/97126882/b31ffc24-40b6-4980-bf32-c1a64e8a2b56)

As you can see the DNS check is good, and it has worked great in the past. Does anybody know how to solve this?
Many thanks in advance!

**Comment by queenofcorgis** [comment]
@flyingcards thanks for the post! 

Have you checked out [this Discussion](https://github.com/orgs/community/discussions/22052)? That may be useful in solving 😃

**Comment by flyingcards** [comment]
Thanks for the help, I was able to solve it.
